topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md

```diff
@@ -0,0 +1,944 @@
+---
+layout: tutorial_hands_on
+
+title: Pox virus genome analysis from tiled-amplicon sequencing data
+subtopic: one-health
+level: Advanced
+questions:
+- Which special challenges does one encounter during sequence data analysis of pox viruses?
+- How can standard workflows for viral mutation calling and consensus generation be adapted to the particularities of pox viruses?
+- How can viral consensus sequences, multiple-sequence alignments and mutation calls be generated from each other and used to answer questions about the data? 
+objectives:
+- Learn how to deal with pox virus genomes inverted terminal repeats through a combination of wet lab protocol and tailored bioinformatics
+- Construct a sample consensus genome from mapped reads
+- Explore a recombinant pox virus genome via a multiple-sequence alignment of consensus genome and references and through lists of mutations derived from it
+time_estimation: 4H
+key_points:
+- Compared to other viruses, analysis of sequencing data from pox viruses is complicated by the rather large genome size and by the presence of inverted terminal repeats, but well-designed wet lab and bioinformatic workflows can handle these particularities.
+- Galaxy and its huge set of tools are flexible enough to handle half-genome sequencing data efficiently.
+- A mapping-based approach can result in high-quality consensus genome reconstructions from pox virus sequencing data and appears promising even when dealing with recombinant samples.
+
+requirements:
+  -
+    type: "internal"
+    topic_name: introduction
+  -
+    type: "internal"
+    topic_name: galaxy-interface
+    tutorials:
+      - collections
+      - upload-rules
+  -
+    type: "internal"
+    topic_name: sequence-analysis
+    tutorials:
+      - quality-control
+      - mapping
+contributors:
+- wm75
+tags:
+- virology
```

Comment:
```suggestion
- virology
- public-health
```

we'll make a public-health synthetic topic soon

Reply:
Hmm, yeah, ok. We can do that. I have an association of "one health" with a specific project (OHP) but if it's a general enough term? then yea we can go with that.

Reply:
https://www.who.int/health-topics/one-health - it's almost turning into a buzzword these days :)

Reply:
aye, fine for me. Updated the suggestion :)